rust/private/rustc.bzl

```diff
@@ -628,7 +628,12 @@ def construct_arguments(
     rustc_flags.add_all(rust_std_paths, before_each = "-L", format_each = "%s")
     rustc_flags.add_all(rust_flags)
 
-    data_paths = getattr(attr, "data", []) + getattr(attr, "compile_data", [])
+    # Deduplicate data paths due to https://github.com/bazelbuild/bazel/issues/14681
+    data_paths = getattr(attr, "data", [])[:]
```

Comment:
What's the `[:]` for?

Reply:
It was to copy, because an attr is immutable, but I like the depset thing so I've done that instead :)